script.js

```diff
@@ -0,0 +1,27 @@
+// one-var
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Definition for rule 'import/extensions' was not found.](https://app.codacy.com/gh/WebHeroSchool/js_test/pullRequest?prid=5502397)